rules/windows/defense_evasion_unusual_ads_file_creation.toml

```
@@ -16,7 +16,66 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Unusual File Creation - Alternate Data Stream"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Unusual File Creation - Alternate Data Stream
+
+Alternate Data Streams (ADS) are file attribute only found on the NTFS file system. In this file system, a file is built
+up from a couple of attributes; One of them is $Data, aka the data attribute.
+
+The regular data stream, also referred to as the unnamed data stream since the name string of this attribute is empty,
+contains the data inside the file. So any data stream that has a name is considered an alternate data stream.
+
+Attackers can abuse these alternate data streams to hide malicious files, string payloads, etc. This rule detects the
+creation of alternate data streams on highly targeted file types.
+
+#### Possible investigation steps
+
+- Retrieve the contents of the alternate data stream, and analyze it for potential maliciousness. Analysts can use the
+following PowerShell cmdlet to accomplish this:
+  - `Get-Content -file C:\\Path\\To\\file.exe -stream ADSname`
+- Investigate the process execution chain (parent process tree) for unknown processes. Examine their executable files
+for prevalence, whether they are located in expected locations, and if they are signed with valid digital signatures.
+- Investigate abnormal behaviors observed by the subject process such as network connections, registry or file
+modifications, and any spawned child processes.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Assess whether this behavior is prevalent in the environment by looking for similar occurrences across hosts.
+- Retrieve the process executable and determine if it is malicious:
+  - Use a private sandboxed malware analysis system to perform analysis.
+    - Observe and collect information about the following activities:
+      - Attempts to contact external domains and addresses.
+      - File and registry access, modification, and creation activities.
+      - Service creation and launch activities.
+      - Scheduled tasks creation.
+  - Use the PowerShell Get-FileHash cmdlet to get the files' SHA-256 hash values.
+    - Search for the existence and reputation of the hashes in resources like VirusTotal, Hybrid-Analysis, CISCO Talos, Any.run, etc.
+
+### False positive analysis
+
+- If this activity is expected and noisy in your environment, consider adding exceptions — preferably with a combination
+of process executable and file conditions.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Isolate the involved host to prevent further post-compromise behavior.
+- If the triage identified malware, search the environment for additional compromised hosts.
```

```suggestion
- If triage identified malware, search the environment for additional compromised hosts.
```
applies elsewhere in the PR

---

Same as https://github.com/elastic/detection-rules/pull/2305/files#r975240781, it doesn't sound wrong to me, as it refers to the process of triaging